server/src/Services/ExhangeRatesManager.php

```diff
@@ -97,12 +97,38 @@ public function convertPrices()
     {
         $result = $this->result;
 
+        // This is a check to see if we're converting the cart total
+        if (isset($result['cart']) && !empty($result['cart']->cart_total)) {
+            return $this->convertPriceCartTotal($result);
+        }
+
+        // Need to ascertain if array of objects or just an object
+        try {
+            if (count($this->result) > 0) {
+                return $this->convertMultiplePrices($result);
+            }
+        } catch (\Throwable $th) {
+            return $this->convertSinglePrice($result);
+        }
+    }
+
+    public function convertPriceCartTotal($result)
+    {
+        // Note, hard coding this base currency for now (USD). TODO: query which currency will be the base, set by
```

Comment:
@AlexeiaW let's store the default currency in config somewhere, and pull it out when the currency is not set.